docs/usage/shoot_maintenance.md

```diff
@@ -40,8 +40,19 @@ spec:
       machineImageVersion: true
 ```
 
-Please refer to [this document](../proposals/05-versioning-policy.md) that describes the exact versioning policy.
+During the daily maintenance, the Gardener Controller Manager updates the Shoot's Kubernetes and machine image version based on the following criteria:
+ - there is a higher version available and the Shoot opted-in for automatic version updates
+ - the currently used version is `expired`
```

Comment:
Could you make it more clear that only one of the criteria is enough to trigger version update?